www/view/layout.html

```diff
@@ -73,22 +73,21 @@
 					<li class=" dropdown">
 						<a href="/HaxeFoundation" data-toggle="dropdown" class="dropdown-toggle">Connect <b class="caret"></b></a>
 						<ul class="dropdown-menu">
-							<li><a href="https://github.com/HaxeFoundation"><i class="fa fa-github"></i> Github</a></li>
-							<li><a href="https://github.com/HaxeFoundation/haxe/issues"><i class="fa fa-github"></i> Bug reports</a></li>
-							<li><a href="https://stackoverflow.com/questions/tagged/haxe"><i class="fa fa-stack-exchange"></i> Stack Overflow</a></li>
-							<li><a href="https://groups.google.com/group/haxelang?hl=en"><i class="fa fa-envelope-o"></i> Google Groups</a></li>
-							<li><a href="https://webchat.freenode.net/?channels=haxe"><i class="fa fa-comments-o"></i> IRC</a></li>
+							<li><a href="https://github.com/HaxeFoundation" rel="external"><i class="fa fa-github"></i> Github</a></li>
+							<li><a href="https://github.com/HaxeFoundation/haxe/issues" rel="external"><i class="fa fa-github"></i> Bug reports</a></li>
+							<li><a href="https://stackoverflow.com/questions/tagged/haxe" rel="external"><i class="fa fa-stack-exchange"></i> Stack Overflow</a></li>
+							<li><a href="http://community.haxe.org/" rel="external"><i class="fa fa-envelope-o"></i> Forums</a></li>
+							<li><a href="https://gitter.im/HaxeFoundation/haxe" rel="external nofollow"><i class="fa fa-comments-o"></i> Chat</a></li>
 							<li><a href="https://haxe.org/blog"><i class="fa fa-rss"></i> Blog</a></li>
 							<li class="divider"></li>
-							<li><a href="https://plus.google.com/communities/103302587329918132234"><i class="fa fa-google-plus"></i> Google+</a></li>
-							<li><a href="https://www.facebook.com/haxe.org/"><i class="fa fa-facebook"></i> Facebook</a></li>
-							<li><a href="https://twitter.com/search?q=%23haxe"><i class="fa fa-twitter"></i> #haxe</a></li>
-							<li><a href="https://twitter.com/haxelang"><i class="fa fa-twitter"></i> &#64;haxelang</a></li>
-							<li><a href="https://twitter.com/haxe_org"><i class="fa fa-twitter"></i> &#64;haxe_org</a></li>
+							<li><a href="https://www.facebook.com/haxe.org/" rel="external"><i class="fa fa-facebook"></i> Facebook</a></li>
+							<li><a href="https://twitter.com/search?q=%23haxe" rel="external"><i class="fa fa-twitter"></i> #haxe</a></li>
+							<li><a href="https://twitter.com/haxelang" rel="external"><i class="fa fa-twitter"></i> @haxelang</a></li>
+							<li><a href="https://twitter.com/haxe_org" rel="external"><i class="fa fa-twitter"></i> @haxe_org</a></li>
```

Comment:
Turn out ufront's template engine doesn't like the 2 `@` signs here. Not sure what's the right way to escape it...

Reply:
Found it, it's `@@`. Just committed a fix.